### ISSUE TITLE
The display of file path is delayed by data preview. 

### ISSUE THREAD

**Issue by vzhuqin** [dotnet/machinelearning-modelbuilder#1879]
**System Information (please complete the following information):**
 - Model Builder Version (available in Manage Extensions dialog): 16.9.1.2152802
 - Microsoft Visual Studio Enterprise 2019: 16.11.5

**Describe the bug**
- On which step of the process did you run into an issue: Data page
- Clear description of the problem: Browse to input a file data set, then find that the file path is displayed only after loaded the data preview. 

**To Reproduce**
1. Select Create a new project from the Visual Studio 2019 start window;
2. Choose the C# Console App (.NET Core) project template with .Net 5.0;
3. Add model builder by right click on the project;
4. Select one scenario support file dataset;
5. On data page, click "Browse..." to input the data set;
6. See that the file path is displayed only after loaded the data preview.

**Expected behavior**
the file path is displayed after input the dataset, should not be affected by the data preview.

**Screenshots**
![File path slow display](https://user-images.githubusercontent.com/81727020/139397249-38571e81-789d-4f9e-a9b5-2d649d501681.gif)

**Comment by beccamc** [comment]
This is lower priority. Adding to the data tab p1/p2 issue and closing for now.
